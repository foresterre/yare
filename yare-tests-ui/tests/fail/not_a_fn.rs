use yare::parameterized;

#[parameterized(
    case1 = { 0 },
)]
struct A;

fn main() {}
