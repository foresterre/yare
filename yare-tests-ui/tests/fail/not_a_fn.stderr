error: expected `fn`
 --> tests/fail/not_a_fn.rs:6:1
  |
6 | struct A;
  | ^^^^^^
